lib/src/ui/checkout_page.dart

```diff
@@ -94,16 +94,22 @@ class _CheckoutPageState extends State<CheckoutPage> {
   void Function() _createAttemptPaymentFunction(BuildContext context, Future<IntentClientSecret> paymentIntentFuture) {
     return () async {
       showProgressDialog(context);
-      final initialPaymentIntent = await paymentIntentFuture.whenComplete(() => hideProgressDialog(context));
+      final initialPaymentIntent = await paymentIntentFuture.catchError((error){
+        debugPrint(error.toString());
+        hideProgressDialog(context);
+      });
```

Comment:
@ezet now even `await paymentIntentFuture` fails, dialog with the loader will be closed

Reply:
But why is the debugPrint there? Shouldn't it be removed?

Reply:
removed